react/features/base/settings/actionTypes.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Create an action for when the settings are updated.
+ *
+ * {
+ *     type: SETTINGS_UPDATED,
+ *     profile: {
+ *         audioOutputDeviceId: string,
+ *         cameraDeviceId: string,
+ *         localFlipX: boolean,
```

Comment:
drive-by comment: is this ever used? can we get rid of it?

Reply:
Yes localFlipX should be used. It shows up on the local video thumbnail when config.enableLocalVideoFlip is true.

Reply:
Yeah, but is there a reason to have it configurable, and stored in the settings, vs have it always on? IIRC there used to be a way to toggle this, but I no longer see that, so it looks like all the machinery to change the setting is there, but there seems to be no way to trigger it? Or am I missing something?

Reply:
Is there a reason to have it configurable? Right now not that I could think of. I recall bringing this up with Emil in September '17. While ironically I remember bringing it up I also remember forgetting to enable it by default like he wanted. I recall him saying it should just be on by default and I think the config dropped. I'd get confirmation from him and move forward.